docs/_sidebar.md

```diff
@@ -139,4 +139,7 @@
     - [Histograms of Oriented Gradients for Human Detection](papers/histograms-of-oriented-gradients-for-human-detection-2005.md)
 
   - 16.Semantic Segmentation
-    - [3D U-Net: Learning Dense Volumetric Segmentation from Sparse Annotation, 2016](papers/3d-u-net-2016.md)
\ No newline at end of file
+    - [3D U-Net: Learning Dense Volumetric Segmentation from Sparse Annotation, 2016](papers/3d-u-net-2016.md)
+  
+  - 17.Deep Learning for Biology
+    - [A^2-Net: Molecular Structure Estimation from Cryo-EM Density Volumes, 2019](papers/a-square-net-Molecular-Structure-Estimation-from-Cryo-EM-Density-Volumes.md)
```

Comment:
Please fix the Upper Case `Molecular-....` in your links to prevent the 404 error.